src/dmd/expressionsem.d

```diff
@@ -5981,6 +5981,13 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             ob.writestring(")");
             ob.writenl();
         }
+        if (global.params.makeDeps && global.params.oneobj)
+        {
+            OutBuffer* ob = global.params.makeDeps;
+            ob.writestringln(" \\");
+            ob.writestring("  ");
+            escapePath(ob, toPosixPath(name));
```

Comment:
Whether `\` must be converted to `/` might depend on what the version of make or what your actual makefile uses, but should be ok for most. `escapePath` escapes `(` and `)`, but that should be unnecessary for `make`, Instead, `$` has to be dealt with.

Reply:
OK. I'm definitely not a make guru. I've seen cases where there is a mix of `\` and `/` in the same file path.
I assumed normalizing everything to `/` would be the safest.
I thought `escapePath` also escape the spaces. These should probably be escaped otherwise we could have broken dependency files.